x-pack/plugins/observability_solution/apm/server/routes/entities/utils/merge_entities.ts

```diff
@@ -27,25 +39,33 @@ export function mergeEntities({
   return [...mergedEntities.values()];
 }
 
-function mergeFunc(entity: ServiceEntities, existingEntity?: MergedServiceEntities) {
+function mergeFunc(entity: EntityLatestServiceRaw, existingEntity?: MergedServiceEntity) {
+  const hasLogMetrics = isFiniteNumber(entity.entity.metrics.logRate)
```

Comment:
since you move the logic here, it would be nice to update the test to cover `hasLogMetrics`

Reply:
https://github.com/elastic/kibana/pull/192086/commits/9842b8ee703b8809bb7074b2b742a2a635841dfc